bin/octokit-schema.js

```diff
@@ -89,4 +133,4 @@ async function run() {
   }
 }
 
-run();
+run().catch(console.error);
```

Comment:
Can we try to handle errors instead of just logging them?

Reply:
This is here because it's otherwise technically a floating promise - otherwise it'll be an uncaught rejection error. I originally added it to make sure I wasn't missing errors when things seemed to just not be working, so it can be removed.